internal/pkg/agent/cmd/install.go

```diff
@@ -84,6 +86,12 @@ would like the Agent to operate.
 func installCmd(streams *cli.IOStreams, cmd *cobra.Command) error {
 	var err error
 
+	if isFleetServerFlagProvided(cmd) {
+		if installServers, _ := cmd.Flags().GetBool(flagInstallServers); !installServers {
+			return fmt.Errorf("to install fleet server `--%s` flag must be used", flagInstallServers)
```

Comment:
Or it just is automatic? Why make the user have to add --install-servers when we know it is needed?

Reply:
i like that.

Reply:
This would make the change non-breaking which is much nicer, just make sure to log that this is happening when we do this.